service/src/test/java/bio/terra/landingzone/stairway/flight/delete/DeleteLandingZoneResourcesStepTest.java

```diff
@@ -49,21 +52,20 @@ public class DeleteLandingZoneResourcesStepTest {
   void setup() {
     workingMap = new FlightMap();
     inputMap = new FlightMap();
-
-    when(landingZoneManagerProvider.createLandingZoneManager(any(LandingZoneTarget.class)))
```

Comment:
It's because in `doStep_failsIfLzNotFound()`, a `LandingZoneNotFoundException` is thrown when it tries to get the landing zone.
Lines 47-54 in `DeleteLandingZoneResourcesStep`:
```
    LandingZoneRecord landingZoneRecord;
    try {
      // Look up the landing zone record from the database
      landingZoneRecord = landingZoneDao.getLandingZoneRecord(landingZoneId);
    } catch (LandingZoneNotFoundException e) {
      logger.error("Landing zone not found. id={}", landingZoneId, e);
      return new StepResult(StepStatus.STEP_RESULT_FAILURE_FATAL, e);
    }
```

This is before it gets to where `flightContext.getWorkingMap()` is called, so it never gets there, and the mocked method is never called.

That's the problem with shared setup like that - sooner or later, you end up testing something that breaks it.

Reply:
Ah good eye, thanks @blakery .